buildspec.yml

```diff
@@ -16,7 +16,12 @@ phases:
     commands:
       - echo Build started on `date`
       - echo Building the Docker image...
-      - python src/main.py --buildspec $FRAMEWORK/buildspec.yml --framework $FRAMEWORK
+      - |
+        if expr "${CODEBUILD_BUILD_ARN}" : ".*tensorflow-1" >/dev/null; then
```

Comment:
can you reply the ${CODEBUILD_BUILD_ARN} full value here.

Reply:
It will be in form of arn:aws:codebuild:us-west-2:669063966089:build/dlc-pr-framework:some-number

Example - arn:aws:codebuild:us-west-2:669063966089:build/dlc-pr-tensorflow-1:884bed12-123a-4398-9f1d-e6921a202b41